profiler/src/TraceAnalyzer.php

```
@@ -111,33 +111,33 @@ private function parseTraceFile($handle): void
                 continue;
             }
 
-            $level = (int)$parts[0];
-            $functionNumber = (int)$parts[1];
+            $level = (int) $parts[0];
+            $functionNumber = (int) $parts[1];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$functionNumber'. (undefined)

(UnusedLocalVariable)